README.md

```diff
@@ -11,6 +11,7 @@ The links below are of the career pages not the main company website. Some of th
 - [BCG Digital Ventures](https://careers.bcgdv.com/locations/sydney/) | Sydney CBD, Sydney | JavaScript, AngularJS, ReactJS, Hadoop, NoSQL
 - [BigCommerce](https://www.bigcommerce.com/careers/) | Sydney CBD, Sydney |  Ruby, PHP, Javascript, Puppet
 - [Canva](https://www.canva.com/careers/) | Sydney CBD, Sydney | Java, Javascript, NodeJs, ReactJs, Typescript
+- [CSIRO’s Data61](https://jobs.csiro.au/search/?createNewAlert=false&q=data61&locationsearch=australia/) | Sydney CBD, Sydney; Melbourne CBD, Melbourne; Brisbane; Canberra | Javascript, NodeJs, Typescript, TerriaJS
```

Comment:
Would you please add the areas of Brisbane and Canberra and also if possible sort them by city name like Bisbane, then Caberra ... Sydney. Thanks!